vm-transfer/config.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+
+class Config(object):
+    SRC_CLUSTER_IP = os.environ.get("SRC_CLUSTER_IP", None)
+    SRC_ACCOUNT = os.environ.get("SRC_ACCOUNT", None)
+    SRC_USERNAME = os.environ.get("SRC_USERNAME", None)
+    SRC_PASSWORD = os.environ.get("SRC_PASSWORD", None)
+    SRC_PROJECT = os.environ.get("SRC_PROJECT", None)
```

Comment:
I would rename this to SRC_PROJECT_ID. It probably seems obvious to you now, but a few months from now, you will not remember if this means project_id or project_name. Same comment in regards to SRC_ACCOUNT (is it supposed to be id or name?) and same comment in regards to the DST constants below.
Or alternatively, if you don't want to rename the constants, at least put a comment in front of each one of them that clarifies what is expected to be in the constant.

Reply:
Following my comment below, I now suggest the new name to be SRC_TRANSFER_PROJECT_ID.